skycoin/skywire
# Is public

Did you run `make format && make check`?
yes

Fixes #	

 Changes:	
- Removed `HTTPResponse` and used `HTTPError` instead 
- Removed `Msg` from `HTTPError` and changed `Err` to `string`
- Deleted `pkg/util/buildinfo/buildinfo.go` and used `buildinfo` from `dmsg` instead

How to test this PR:
1. Make `is_public:true` in config
2. Change `log_level` to `debug` in config
3. Run visor on test `./skywire-visor skywire-config`
4. Check if the visor is registered on `service-discovery`
